src/main/webapp/view/transcript/transcript.js

```diff
@@ -112,7 +112,13 @@ function deleteTranscript() {
 /** Seeks transcript to {@code currentTime}, which is given in seconds. */
 function seekTranscript(currentTime) {
   const currentTimeMs = window.secondsToMilliseconds(currentTime);
-  if (currentTimeMs <= currentTranscriptLine.endTimestampMs) {
+  if (currentTimeMs < currentTranscriptLine.startTimestampMs) {
```

Comment:
Will `currentTranscriptLine` be `undefined` at first?

Reply:
.

Reply:
I'm guessing it would be undefined if seekTranscript is called before the transcript finishes loading. 
But once the lines are added to the DOM, it shouldn't be undefined because currentTranscript is assigned when the lines are added to the DOM. 

https://github.com/googleinterns/zoomtube/blob/a8387b467ebd12eff34340f1d6349833d9be4053/src/main/webapp/view/transcript/transcript.js#L83

Reply:
Ok, thanks for adding an issue to track this

Reply:
Of course, thanks for bring this up! For reference, the issue created is : #211